Earthfile

```diff
@@ -1,21 +1,33 @@
-FROM node:12
+VERSION 0.6
 
-build:
-    COPY package.json package-lock.json ./
-    RUN yarn install
-    SAVE ARTIFACT node_modules
-    COPY src src
-    RUN mkdir -p ./dist
-    COPY postcss.config.js .
-    COPY tailwind.config.js .
-    COPY webpack.config.js .
-    ARG GIT_SHA
-    RUN REPLAY_BUILD_VISUALIZE=1 NODE_ENV=production ./node_modules/.bin/webpack --env REPLAY_RELEASE=$GIT_SHA
-    SAVE ARTIFACT dist /dist AS LOCAL ./dist
+# NOTE(dmiller): this needs to be 16._15_ otherwise playwright will fail to install, at least until we update our playwright version
+FROM node:16.15-bullseye-slim
 
-dist:
-    COPY +build/node_modules node_modules
-    COPY index.html .
-    COPY +build/dist dist
-    RUN tar -czf dist.tgz index.html dist node_modules
-    SAVE ARTIFACT dist.tgz AS LOCAL ./dist.tgz
\ No newline at end of file
+image:
+  RUN apt-get update && apt-get install -y python3 build-essential
+  WORKDIR /app
+
+  COPY package.json yarn.lock .
+  RUN yarn set version 3.2.1
+  COPY packages/protocol/package.json packages/protocol/package.json .
+  COPY packages/bvaughn-architecture-demo/package.json packages/bvaughn-architecture-demo/package.json .
+  COPY packages/design/package.json packages/design/package.json .
+  COPY packages/bvaughn-architecture-demo/playwright/package.json packages/bvaughn-architecture-demo/playwright/package.json .
+
+  COPY .yarnrc.yml .
+  COPY .yarn/plugins/@yarnpkg/plugin-workspace-tools.cjs .yarn/plugins/@yarnpkg/plugin-workspace-tools.cjs
```

Comment:
No idea if this is the right thing to do to get this yarn setup working in a container. @markerikson any ideas?

Reply:
Very likely, yeah. You'll also need to make sure you copy the actual Yarn "binary" file, currently `.yarn/release/yarn-3.2.1.cjs`